lib/buildingsync/makers/workflow_maker_phase_zero.rb

```diff
@@ -256,7 +268,7 @@ def configure_for_scenario(osw, scenario)
           if measure_category == 'Other HVAC'
 
             # Other HVAC / OtherHVAC
-            measure_name = measure.elements["#{@ns}:TechnologyCategories/#{@ns}:TechnologyCategory/#{@ns}:OtherHVAC/#{@ns}:MeasureName"].text
+            measure_name = measure.elements["#{@ns}:TechnologyCategories/#{@ns}:TechnologyCategory/#{@ns}:*/#{@ns}:MeasureName"].text
```

Comment:
this seems very heavy if/then/else type code. I am okay for now leaving this, but this does not seem sustainable. I suggest trying to DRY up this code by moving the measure_category to a hash of some sort.

Reply:
see issue #33